src/parser/jobs/drk/modules/ResourceSimulator.js

```diff
@@ -24,23 +24,30 @@ const BLOODSPILLER_BLOOD_COST = 50
 // Meters
 // ------
 const MAX_MANA = 9480
+const MANA_REGEN_PERCENT_PER_TICK = 0.02
+const TICK_RATE = 3000
+
 const MAX_BLOOD = 100
 //const MANA_PER_OUT_OF_COMBAT_TICK = 568 // DA should be used at least 2 ticks pre pull
 //const DARKSIDE_MANA_COST = 600
 
-const BLOOD_PRICE_BLOOD_DAMAGE_TRIGGERED_BLOOD_AMOUNT = 1
-const BLOOD_PRICE_BLOOD_DAMAGE_TRIGGERED_MANA_AMOUNT = 120
+const BLOOD_PRICE_DAMAGE_TAKEN_BLOOD_AMOUNT = 1
+const BLOOD_PRICE_DAMAGE_TAKEN_MANA_AMOUNT = 120
 const BLOOD_PRICE_BLOOD_PASSIVE_RATE = 3000
 const BLOOD_PRICE_BLOOD_PASSIVE_AMOUNT = 4
 const BLOOD_PRICE_MAX_DURATION = 15000
 
+const CARVE_AND_SPIT_NO_DA_MANA_GAIN = 1200
+
 //using this as a lookup table instead of a collector
 const RESOURCE_STATUS_EFFECTS = {
 	[STATUSES.ANOTHER_VICTIM.id]: {
 		duration: 15000,
```

Comment:
This will still trip the linter I think. I disabled magic numbers for this whole const declaration.

Reply:
Didn't trip it to do individual lines on my testing.  Let me know which style is preferred.

Reply:
Numbers as literal values in objects shouldn't trip the linter - the magic number rule is to prevent unnamed numbers, and the object key acts as a name.